azure-cpi.html.md.erb

```diff
@@ -115,7 +117,7 @@ Example of 10GB disk:
 ```yaml
 disk_pools:
 - name: default
-  disk_size: 10_240
+  disk_size: 10_240 # On Azure the disk size must be in the range [1 * 1024, 1023 * 1024].
```

Comment:
lets add that info above then so that its all in one place.

Sent from my iPhone

> On Jul 13, 2016, at 6:15 PM, Chou Hu <notifications@github.com> wrote:
> 
> In azure-cpi.html.md.erb:
> 
> > @@ -115,7 +117,7 @@ Example of 10GB disk:
> >  ```yaml
> >  disk_pools:
> >  - name: default
> > -  disk_size: 10_240
> > +  disk_size: 10_240 # On Azure the disk size must be in the range [1 * 1024, 1023 * 1024].
> Above only mentions for the size of ephemeral_disk. And it does not mention that the size must be larger than 1024 MB. So we think we should add a comment here.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>